mne/gui/_coreg.py

```diff
@@ -126,10 +122,10 @@ def __init__(self, info_file, subject=None, subjects_dir=None,
                  fiducials='auto', head_resolution=None,
                  head_opacity=None, hpi_coils=None,
                  head_shape_points=None, eeg_channels=None, orient_glyphs=None,
-                 scale_by_distance=None, project_eeg=None, mark_inside=None,
+                 scale_by_distance=None, mark_inside=None,
```

Comment:
Or maybe earlier in the list? Not sure
```suggestion
                 scale_by_distance=None, *, mark_inside=None,
```